crashbin_app/views.py

```diff
@@ -164,7 +163,7 @@ class _ButtonInfo:
 
 def _get_settings(request: HttpRequest, pk: int, setting: str) -> HttpResponse:
     Element = typing.Union[User, Label, Bin]
-    all_elements: QuerySet
+    all_elements: typing.Iterable[Element]
```

Comment:
This seems wrong - looking at the code, `all_elements` is a `QuerySet` and not a Python iterable?